src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -462,6 +462,23 @@ class HttpGoogleServicesDAO(
     result
   }
 
+  def diagnosticBucketRead(user: RawlsUser, bucketName: String): Future[Option[ErrorReport]] = {
+    getUserCredentials(user) map { credentialOpt =>
```

Comment:
this is done on a user's request, don't we already have the user's access token? If we do, I think we should use that.